unattended_scripts/install_functions/opendistro/common.sh

```diff
@@ -67,13 +71,13 @@ installPrerequisites() {
     logger "Installing all necessary utilities for the installation."
 
     if [ ${sys_type} == "yum" ]; then
-        eval "yum install curl unzip wget libcap tar -y ${debug}"
+        eval "yum install curl unzip wget libcap tar gnupg openssl -y ${debug}"
     elif [ ${sys_type} == "zypper" ]; then
         eval "zypper -n install curl unzip wget ${debug}"         
-        eval "zypper -n install libcap-progs tar ${debug} || zypper -n install libcap2 tar ${debug}"
+        eval "zypper -n install libcap-progs tar gnupg openssl ${debug} || zypper -n install libcap2 tar gnupg openssl ${debug}"
     elif [ ${sys_type} == "apt-get" ]; then
         eval "apt-get update -q $debug"
-        eval "apt-get install apt-transport-https curl unzip wget libcap2-bin tar -y ${debug}"
+        eval "apt-get install apt-transport-https curl unzip wget libcap2-bin tar gnupg openssl -y ${debug}"
```

Comment:
In wazuh-cert-tool.sh file:

- Add a method to check if openssl is installed using `command -v openssl`, this method should return 1 (installed) or 0 (not installed).
- Add a method in the main that checks the return value of the previous method
  - Returned value is 1: continue
  - Returned value is 0: error message and exit

In common.sh file:

- Before install the prerequisites, call the wazuh-cert-tool.sh file method to check if openssl is installed, if not, add to variable the string: `openssl`
  - openssl="openssl"
- Add this new var to install command, if its empty, it will not install openssl, if has the openssl value, it will install openssl
  - .... install .... ${openssl} -y ...

Reply:
Something like this was implemented.